modules/lang-painless/src/main/antlr/PainlessParser.g4

```diff
@@ -156,11 +156,11 @@ postdot
     ;
 
 callinvoke
-    : DOT DOTID arguments
+    : nullSafe=COND? DOT DOTID arguments
```

Comment:
I strongly prefer a comment over a new variable.  This keeps consistency with everything else in the Walker.